src/routes/blog/post/everyone-can-do-devrel-but-should-they/+page.markdoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Everyone can do DevRel (but should they?)
+description: Exploring the question if everyone should be rushing towards a job in Developer Relations?
+date: 2024-01-07
+cover: /images/blog/everyone-can-do-devrel-but-should-they/cover.png
+timeToRead: 5
+author: aditya-oberai
+category: culture
```

Comment:
I don't think this would be the appropriate category

It might make sense to create a new one called `devrel` or `community`